.github/workflows/release.yml

```diff
@@ -155,24 +145,94 @@ jobs:
         run: ci/cli.sh
       - name: install
         run: ci/install.sh
-        if: ${{ matrix.upgrade == false }}
-      - name: install last release
-        run: ci/install_last_release.sh
-        if: ${{ matrix.upgrade == true }}
       - name: wait
         run: sleep 600
-      - name: before version upgrade/downgrade setup and test
+      - name: test
+        run: ci/test.sh
+      - name: debug
+        run: ci/debug.sh
+        if: always()
+      - name: uninstall
+        run: ci/uninstall.sh
+        if: always()
+  ci-downgrage:
+    if: github.repository == 'convox/convox'
+    environment: 'UpgradeDowngradeTests'
```

Comment:
Can `UpgradeDowngradeTests` be renamed? In case we want to add more additional tests...

Reply:
this environment is configured by @heronrs, i don't have access. He can give more context on it

Reply:
Yeah sorry, that was a bad name. I can't rename the environment sadly but I'll create a new one later. Maybe name it just ManuApprovalTests.

Merging for now just so we can move forward